source/api_documentation/network_integration/promo_numbers/_put_promo_number.rst

```diff
@@ -53,3 +53,67 @@
   Response Body:
 
   Same as a GET response, includes all the promo number properties.
+
+  .. raw:: html
+
+    <hr>
+
+  Moving an Advertiser Campaign Promo Number
+
+  **Note:** When a promo number is moved, its **id_from_network** will be automatically adjusted.
```

Comment:
🔧 Will communicate in [this slack discussion](https://invoca.slack.com/archives/C05N0QGR66Q/p1706114662337359) to confirm our next steps forward